store-implementation/accumulo-store/src/main/java/uk/gov/gchq/gaffer/accumulostore/MiniAccumuloStore.java

```diff
@@ -0,0 +1,274 @@
+        addOperationHandler(DeleteNamedView.class, null);
+        addOperationHandler(Max.class, null);
+        addOperationHandler(Min.class, null);
+        addOperationHandler(Sort.class, null);
+        addOperationHandler(OperationChain.class, null);
+        addOperationHandler(OperationChainDAO.class, null);
+        addOperationHandler(ValidateOperationChain.class, null);
+        addOperationHandler(GetWalks.class, null);
+        addOperationHandler(GenerateElements.class, null);
+        addOperationHandler(GenerateObjects.class, null);
+        addOperationHandler(Validate.class, null);
+        addOperationHandler(Count.class, null);
+        addOperationHandler(CountGroups.class, null);
+        addOperationHandler(Limit.class, null);
+        addOperationHandler(DiscardOutput.class, null);
+        addOperationHandler(GetSchema.class, null);
+        addOperationHandler(uk.gov.gchq.gaffer.operation.impl.Map.class, null);
+        addOperationHandler(If.class, null);
+        addOperationHandler(While.class, null);
+        addOperationHandler(ForEach.class, null);
+        addOperationHandler(ToSingletonList.class, null);
+        addOperationHandler(Reduce.class, null);
+        addOperationHandler(Join.class, null);
+        addOperationHandler(CancelScheduledJob.class, null);
+        addOperationHandler(SetVariable.class, null);
+        addOperationHandler(GetVariable.class, null);
+        addOperationHandler(GetVariables.class, null);
+        addOperationHandler(Filter.class, null);
+        addOperationHandler(Transform.class, null);
+        addOperationHandler(Aggregate.class, null);
+        addOperationHandler(AddSchemaToLibrary.class, null);
+        addOperationHandler(AddStorePropertiesToLibrary.class, null);
+        addOperationHandler(GetTraits.class, null);
+        addOperationHandler(AddElementsFromHdfs.class, null);
+        addOperationHandler(GetElementsBetweenSets.class, null);
+        addOperationHandler(GetElementsWithinSet.class, null);
+        addOperationHandler(SplitStoreFromFile.class, null);
+        addOperationHandler(SplitStoreFromIterable.class, null);
+        addOperationHandler(SplitStore.class, null);
+        addOperationHandler(SampleElementsForSplitPoints.class, null);
+        addOperationHandler(SampleDataForSplitPoints.class, null);
+        addOperationHandler(ImportAccumuloKeyValueFiles.class, null);
+        addOperationHandler(SummariseGroupOverRanges.class, null);
+        addOperationHandler(GetElementsInRanges.class, null);
+    }
+
+    OperationHandler getOperationHandlerExposed(final Class<? extends Operation> opClass) {
+        return super.getOperationHandler(opClass);
+    }
+
+    private void closeMiniAccumuloStore() {
+        if (null == miniAccumuloCluster) {
+            return;
+        }
+        try {
+            miniAccumuloCluster.stop();
+        } catch (final IOException | InterruptedException e) {
+            // Ignore any errors here;
```

Comment:
It would be better if these errors were logged in case they hide any underlying bugs.

Reply:
OK.
Where and at what level(s) would you like them to be logged or would simply not catching them allow them to trigger any further required actions in the builds and pipeline?

Reply:
We tend to use slf4j logger factories: 
```java
private static final Logger LOGGER = LoggerFactory.getLogger(MiniAccumuloStore.class);
// ...
LOGGER.error(myerror)
```

For something like this, you could retry it a couple of times and if it continues to fail, throw the exception.

Reply:
Changed as requested.